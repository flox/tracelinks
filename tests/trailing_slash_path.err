tracelinks: extra trailing characters: trailing/path
