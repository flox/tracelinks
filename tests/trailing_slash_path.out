./tests/trailing_slash_path -> ../tests/file/trailing/path
  ./tests/../tests/file: regular file
